CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ find_package(OpenSSL REQUIRED)
 #get_filename_component(rootPath "${OPENSSL_CRYPTO_LIBRARY}" DIRECTORY)
 #message(STATUS "OpenSSL root: " ${rootPath})
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+	find_library(ISAL isa-l_crypto_static HINTS "${CMAKE_SOURCE_DIR}/ISA-L/Debug" REQUIRED)
```

Comment:
Should this ISA-L lib file go under "lib" (lib\os\windows) directory?

Reply:
I think it should remain outside.  I haven't tested it inside the libs folder. But in openzfs libs folder cmakelist will add some specific properties for building the openzfs libraries and this is already built external library. In zfsin also it seems to be separate. May be @arun-kv  can also comment more on it.

Reply:
I tried building the code and tested basic encryption commands by putting the ISA-L library in openzfs\lib directory. Since we are linking it with libicp, I think we can put it in lib directory.